src/forms/RadioButtonGroup.jsx

```diff
@@ -24,50 +23,27 @@ export default class RadioButtonGroup extends PureComponent {
 		}
 	}
 
-	handleChange(event) {
-		const { onChange } = this.props;
+	handleChange(e) {
+		this.props.onChange && this.props.onChange(e);
 
 		this.setState({
-			selectedValue: event.target.value,
+			selectedValue: e.target.value,
 		});
-
-		if (onChange) {
-			onChange(event);
-		}
 	}
 
 	render() {
-		const {
-			direction,
-			switchDirection,
-			children,
-			name,
-			className,
-			wrap,
-			justifyItems,
-		} = this.props;
-
-		const switchDirectionAttr = switchDirection ? { switchDirection } : '';
+		const { children, className } = this.props;
```

Comment:
Do we need to explicitly use `className` here or `...other` would be better and would carry other props as well?
We don't seem to do anything special with `className` but just apply it to the outmost component, right?

Reply:
I guess technically that would change the original interface, which doesn't support arbitrary props, but I like the idea

Reply:
I also think that using `..other` is better, but I'm going to leave it as is to limit the changes to the interface / hopefully fewer bugs introduced.